scalameta/semanticdb-scalac/src/test/scala/scala/meta/tests/SemanticSuite.scala

```diff
@@ -248,14 +248,13 @@ class SemanticSuite extends DatabaseSuite(SemanticdbMode.Slim) {
       |  CommandeerDSL(null.asInstanceOf[Foo])
       |}
     """.trim.stripMargin,
-    """|[324..324): *.apply[Foo, FooDSL]
+    """|[324..324): *.apply[Foo, FooDSL](*)(g.Foo.fooDSL)
```

Comment:
How about we limit this change only to `.foreach/.flatMap/.map`? I think we should try to limit the number of `*` placeholder when possible since multiple `*` make it harder to disambiguate what they are placeholding.